### PR TITLE
feat(mnq): mnq doc updates for ga

### PR DESCRIPTION
**Pull request by RoRoJ**
All M&Q doc updates for GA

### CODE REVIEW

**Comment by derkens-scw** [comment]
https://scaleway-docs-staging.s3-website.fr-par.scw.cloud/en/docs/serverless/messaging/api-cli/connect-aws-cli/

Update this for the no namespace archi. I think there is no new stuff to right besides changing title to "Connecting SQS/SNS to the AWS CLI"

@JulienVdG @ahamez Can you check please 🙏

**Comment by derkens-scw** [comment]
Screenshot 2023-10-10 at 14 31 15
Typo : divded > divided

**Comment by derkens-scw** [comment]
Screenshot 2023-10-10 at 14 34 18

From the Quickstart page, "that you can deactivate IT at any time" ? Proposition as it seems something is missing.

**Comment by derkens-scw** [comment]
Screenshot 2023-10-10 at 14 36 25

From the quickstart create a queue, the + button is not green but purple. Maybe I missing the proper journey though

**Comment by derkens-scw** [comment]
Screenshot 2023-10-10 at 14 42 32

Credentials pabel ? Is it label ?

**Comment by ahamez** [comment]
> Screenshot 2023-10-10 at 14 46 59 We should update with the PHP Links What do you think @ahamez ?

Yes, good idea 👍

File: serverless/containers/how-to/add-trigger-to-a-container.mdx [hunk 1]
```diff
@@ -52,7 +52,7 @@ The configuration of the queue retention may affect the behavior of the trigger.
 3. Click the name of the container you want to manage. The container overview displays.
 4. Click the **Triggers** tab. The trigger configuration displays.
 5. Click **Create new Trigger**. The trigger creation form displays in a pop-up.
-6. Enter a **name** for your trigger and select the **SQS (Scaleway)** type. Select the messaging namespace to use from the drop-down name and enter your queue name.
+6. Enter a **name** for your trigger and select the **SQS (Scaleway)** type, then select your queue name.
 7. Click **Create trigger** to launch the trigger creation.
```
Comment (nerda-codes) [inline serverless/containers/how-to/add-trigger-to-a-container.mdx:57]: Should the "T" for "trigger" in "Create new Trigger" in step 5 be capitalized?

File: serverless/functions/how-to/add-trigger-to-a-function.mdx [hunk 2]
```diff
@@ -42,7 +42,7 @@ The configuration of the queue retention may affect the behavior of the trigger.
 3. Click the name of the function you want to manage. The function overview displays.
 4. Click the **Triggers** tab. The trigger configuration displays.
 5. Click **Create new Trigger**. The trigger creation form displays in a pop-up.
```
Comment (nerda-codes) [inline serverless/functions/how-to/add-trigger-to-a-function.mdx:44]: same comment as above

File: serverless/messaging/how-to/migrate-to-new-mnq.mdx [hunk 3]
```diff
@@ -0,0 +1,78 @@
+---
+meta:
+  title: How to get migrate to the new Messaging and Queuing version
```
Comment (ldecarvalho-doc) [inline serverless/messaging/how-to/migrate-to-new-mnq.mdx:3]:
```suggestion
  title: How to migrate to the new Messaging and Queuing version
```

File: serverless/messaging/how-to/migrate-to-new-mnq.mdx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  - You were using Scaleway Messaging and Queuing before 16 October 2023
+</Message>
+
+## NATS
+
+NATS namespaces and credentials created with Scaleway Messaging and Queuing before October 16 2023 **have been automatically migrated to the new version**.
+
+1. Log in to the [Scaleway console](https://console.scaleway.com/) and click **Messaging** in the side menu, then click **NATS**. You see that your NATS namespaces have now become [NATS accounts](/serverless/messaging/concepts/#nats-account). The credentials of your namespaces are now credentials of your NATS accounts.
+
+2. **Update your endpoints.** The endpoint of your Scaleway Messaging and Queuing NATS service has changed. The new endpoint is displayed in the console. **You must update the endpoint in any code or applications using these endpoints.** After 30 days, the old endpoints will no longer be supported.
```
Comment (ldecarvalho-doc) [inline serverless/messaging/how-to/migrate-to-new-mnq.mdx:33]: maybe add this part to a message box? ` **You must update the endpoint in any code or applications using these endpoints.** After 30 days, the old endpoints will no longer be supported.`

File: serverless/messaging/how-to/migrate-to-new-mnq.mdx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+2. [Activate the SQS protocol](/serverless/messaging/how-to/get-started/#how-to-activate-sqs-or-sns).
+
+    You see that your previous namespaces and queues are not present in the console. This is because the console now uses the new version of Messaging and Queuing, and you must recreate your SQS resources.
+
+3. [Create new credentials](/serverless/messaging/how-to/create-credentials/) for your SQS service.
+
+4. [Create new queues](/serverless/messaging/how-to/create-manage-queues/#how-to-create-a-queue) for your SQS service.
+
+5. **Update your code.** The endpoint of your Scaleway Messaging and Queuing SQS service has changed. The new endpoint is displayed in the console. **You must update the endpoint in any code or applications using these endpoints.** After 30 days, the old endpoints will no longer be supported.
```
Comment (ldecarvalho-doc) [inline serverless/messaging/how-to/migrate-to-new-mnq.mdx:51]: same comment as above

File: serverless/messaging/how-to/migrate-to-new-mnq.mdx [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+1. Log in to the [Scaleway console](https://console.scaleway.com/) and click **Messaging** in the side menu.
+
+2. [Activate the SNS protocol](/serverless/messaging/how-to/get-started/#how-to-activate-sqs-or-sns).
+
+    You see that your previous namespaces and credentials are not present in the console. This is because the console now uses the new version of Messaging and Queuing, and you must recreate your SNS resources.
+
+3. [Create new credentials](/serverless/messaging/how-to/create-credentials/) for your SNS service.
+
+4. **Update your code.** The endpoint of your Scaleway Messaging and Queuing SNS service has changed. The new endpoint is displayed in the console. **You must update the endpoint in any code or applications using these endpoints.** After 30 days, the old endpoints will no longer be supported.
```
Comment (ldecarvalho-doc) [inline serverless/messaging/how-to/migrate-to-new-mnq.mdx:72]: same comment

File: serverless/messaging/how-to/migrate-to-new-mnq.mdx [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+2. [Activate the SNS protocol](/serverless/messaging/how-to/get-started/#how-to-activate-sqs-or-sns).
+
+    You see that your previous namespaces and credentials are not present in the console. This is because the console now uses the new version of Messaging and Queuing, and you must recreate your SNS resources.
+
+3. [Create new credentials](/serverless/messaging/how-to/create-credentials/) for your SNS service.
+
+4. **Update your code.** The endpoint of your Scaleway Messaging and Queuing SNS service has changed. The new endpoint is displayed in the console. **You must update the endpoint in any code or applications using these endpoints.** After 30 days, the old endpoints will no longer be supported. 
+
+If you use the API to manage your resources, you should use the new `v1beta1` version of the Messaging and Queuing SQS API going forward. This is available on the [Scaleway developers website](https://www.scaleway.com/en/developers/api/). Note that your previous resources (SNS namespaces, credentials etc) still exist on the old `v1alpha1` version of the API, which you can also access, but after 30 days these resources will be deleted.
```
Comment (ldecarvalho-doc) [inline serverless/messaging/how-to/migrate-to-new-mnq.mdx:74]: here too, maybe all info after "note that" should go in a note box

File: serverless/messaging/how-to/delete-namespace.mdx [hunk 8]
```diff
@@ -1,43 +1,16 @@
 ---
 meta:
-  title: How to delete a Messaging and Queuing Messaging and Queuing
+  title: How to delete a Messaging and Queuing Messaging and Queuing namespace
   description: This page explains how to delete a Messaging namespace
```
Comment (nerda-codes) [inline serverless/messaging/how-to/delete-namespace.mdx:4]:
```suggestion
  description: This page explains how to delete a Messaging and Queuing namespace
```

File: serverless/messaging/reference-content/nats-overview.mdx [hunk 9]
```diff
@@ -19,15 +19,15 @@ NATS (**N**eural **A**utonomic **T**ransport **S**ystem) is a messaging system t
 
 NATS is a [Cloud Native Computing Foundation](https://www.cncf.io/) project supported by a strong community, and can also be integrated in a Kubernetes Cluster.
 
-## NATS namespaces
+## NATS accounts
 
-You can easily create a NATS namespace from the [Scaleway console](/serverless/messaging/how-to/create-namespace), via the [Scaleway API](https://www.scaleway.com/en/developers/api/messaging-and-queuing/) or via [Terraform](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/guides/mnq_with_nats_terraform_provider).
+You can easily create a NATS account from the [Scaleway console](/serverless/messaging/how-to/get-started/#how-to-create-a-nats-account), via the [Scaleway API](https://www.scaleway.com/en/developers/api/messaging-and-queuing/) or via [Terraform](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/guides/mnq_with_nats_terraform_provider).
```
Comment (nerda-codes) [inline serverless/messaging/reference-content/nats-overview.mdx:24]: I would expect to be redirected to the Scaleway console if the clickable text is "Scaleway console", maybe move the link to "create a NATS account"?

File: serverless/messaging/reference-content/sns-overview.mdx [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+categories:
+  - serverless
+dates:
+  validation: 2023-10-05
+  posted: 2023-01-04
+---
+
+## What is SNS?
+
+SNS is a managed messaging service developed by AWS and supported by Scaleway Messaging and Queuing. It is often coupled to work with [SNS](/serverless/messaging/reference-content/sns-overview/).
```
Comment (nerda-codes) [inline serverless/messaging/reference-content/sns-overview.mdx:18]: Does "It" refer to SNS or Messaging and Queuing? Or should it be "It is often coupled to work with SQS" instead?

File: serverless/messaging/reference-content/nats-overview.mdx (hunk 9 above)
Comment (RoRoJ) [reply serverless/messaging/reference-content/nats-overview.mdx]: Modified

File: serverless/functions/how-to/add-trigger-to-a-function.mdx [hunk 11]
```diff
@@ -41,8 +41,8 @@ The configuration of the queue retention may affect the behavior of the trigger.
 2. Click the relevant functions namespace.
 3. Click the name of the function you want to manage. The function overview displays.
 4. Click the **Triggers** tab. The trigger configuration displays.
-5. Click **Create new Trigger**. The trigger creation form displays in a pop-up.
-6. Enter a **name** for your trigger and select the **SQS (Scaleway)** type. Select the messaging namespace to use from the drop-down name and select your queue name.
+5. Click **Create trigger**. The trigger creation form displays in a pop-up.
+6. Enter a **name** for your trigger and select the **SQS (Scaleway)** type, then select your queue name.
```
Comment (SamyOubouaziz) [inline serverless/functions/how-to/add-trigger-to-a-function.mdx:45]:
```suggestion
6. Enter a **name** for your trigger and select the **SQS** type, then select your queue name.
```

File: serverless/containers/how-to/add-trigger-to-a-container.mdx [hunk 12]
```diff
@@ -51,8 +51,8 @@ The configuration of the queue retention may affect the behavior of the trigger.
 2. Click the relevant containers namespace.
 3. Click the name of the container you want to manage. The container overview displays.
 4. Click the **Triggers** tab. The trigger configuration displays.
-5. Click **Create new Trigger**. The trigger creation form displays in a pop-up.
-6. Enter a **name** for your trigger and select the **SQS (Scaleway)** type. Select the messaging namespace to use from the drop-down name and enter your queue name.
+5. Click **Create trigger**. The trigger creation form displays in a pop-up.
+6. Enter a **name** for your trigger and select the **SQS (Scaleway)** type, then select your queue name.
```
Comment (SamyOubouaziz) [inline serverless/containers/how-to/add-trigger-to-a-container.mdx:55]:
```suggestion
6. Enter a **name** for your trigger and select the **SQS** type, then select your queue name.
```

**Pull request merged**